src/sentry/api/api_owners.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+
+
+class ApiOwner(Enum):
+    """
+    Used to track ownership of APIs
+    Value should map to team's github group
+    """
+
+    DISCOVER_N_DASHBOARDS = "discover-n-dashboards"
+    TELEMETRY_EXPERIENCE = "telemetry-experience"
+    ENTERPRISE = "enterprise"
+    SECURITY = "security"
+    HYBRID_CLOUD = "hybrid-cloud"
+    ISSUES = "issues"
+    PERFORMANCE = "performance"
+    TEAM_STARFISH = "team-starfish"
+    PROFILING = "profiling"
+    GROWTH = "growth"
```

Comment:
There is no growth team currently.

Reply:
yeah, I'm working on that. The list is from CODEOWNERS and there is code owned by growth that's why it shows up here.